pubspec.yaml

```diff
@@ -23,6 +23,7 @@ dependencies:
   path: ^1.8.1
   cached_network_image: ^3.2.1
   pinch_zoom: ^1.0.0
+  flutter_svg: ^1.1.4
```

Comment:
The sample is not running on iOS. Can you check?

Reply:
![image](https://user-images.githubusercontent.com/14128784/185664854-46d933bd-daac-4da4-ac93-98815a35d9ef.png)

here it works, what is your error?

Reply:
Here it works as well.